pkg/controller/migration/convert/typha.go

```diff
@@ -0,0 +1,43 @@
+package convert
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorv1 "github.com/tigera/operator/api/v1"
+	"github.com/tigera/operator/pkg/controller/utils"
+)
+
+func checkTypha(c *components, _ *operatorv1.Installation) error {
+	if c.typha != nil {
```

Comment:
nit, you can de-indent this if you return
```suggestion
	if c.typha == nil {
	  return nil
	}
```